scripts/memegle_scrapper.py

```diff
@@ -2,36 +2,56 @@
 import argparse
 from os.path import exists, isdir, isfile, join, abspath
 from os import mkdir
+import random
 
 parser = argparse.ArgumentParser()
 parser.add_argument('-o', '--output_dir', default='../data/original_images')
 parser.add_argument('-i', '--start_index', default='0')
+parser.add_argument('-n', '--num_pics', default='0')
+parser.add_argument('-r', '--random_pics', default='no')
```

Comment:
Boolean args can be parsed like this, so that user can simply input `-r` instead of `-r yes` for random sampling

https://github.com/Memegle/memegle/blob/9d1712f2565a8a457a0c41456929574e91db395e/server/scripts/migrate.py#L26